### ISSUE TITLE
Why is nobody updating SafariStand?

### ISSUE THREAD

**Issue by jason-hub** [anakinsk/SafariStand#88]
Safari is HORRIBLE without SafariStand.  Why is nobody updating SafariStand?  Is it a money issue?

**Comment by w0lfschild** [comment]
Yes and the macOS plugin dev community is basically me + a few occasional one off devs.

**Comment by vikasbhatia** [comment]
I just wanted to say thank you for all your work @w0lfschild and @anakinsk.

I used SafariStand for the favicons until Apple introduced them. I used Flashlight until they killed a whole bunch of things with SIP. Even though I don't have them any more, I got a ton of use out of them both and I didn't want all your efforts to go unrecognised.

I understand the security standpoint from which Apple's limiting this stuff, but honestly this community can build so many amazing enhancements if they were just given a way to. ¯\\\_(ツ)\_/¯
